src/test/java/oshi/SystemInfoTest.java

```diff
@@ -225,6 +226,27 @@ public void testFileSystem() throws IOException {
         }
     }
 
+       /**
+     * Test disks extraction.
+     *
+     * @throws IOException
+     *             Signals that an I/O exception has occurred.
+     */
+    @Test
+    public void testDisks() throws IOException {
+        SystemInfo si = new SystemInfo();
+        HardwareAbstractionLayer hal = si.getHardware();
+        if (hal.getDisksStores().length > 1) {

```

Comment:
Change this conditional to iterate with a counter from 0 to length, and check the counter rather than 0 in the lines below.